packages/calcite-components/src/components/slider/slider.scss

```diff
@@ -140,15 +158,15 @@
   .handle-extension {
     @apply w-0.5;
     block-size: var(--calcite-slider-handle-extension-height);
-    background-color: var(--calcite-color-text-3);
+    background-color: var(--calcite-slider-handle-extension-color, var(--calcite-color-text-3));
   }
 
   &:hover {
     .handle {
       box-shadow: 0 0 0 3px var(--calcite-color-brand) inset;
```

Comment:
This line, line 169, line 178, line 242, and line 250 should all have a shared token

Reply:
Thanks, for the shared token since it spans different elements and even states, what would be a good name be something like? Something like `--calcite-slider-accent-color`? 

Also wondering about the extension tokens (line 169 and 178) being shared across states, the way it works without tokens is that extension is a separate color then gets highlighted with `--calcite-color-brand` on hover and focus.

Reply:
yes I think `-accent-color-press` is correct here.